src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/CalculateFlowLatencyService.java

```diff
@@ -46,8 +46,9 @@ public void handleCalculateFlowLatencyRequest(String flowId, FlowDirection direc
                 .filter(request -> request.getFlowId().equals(flowId) && request.getDirection() == direction)
```

Comment:
As for me 2 map for each type of searches is better and simplier to manage.

Reply:
Didn't get it. You propose to use two duplicate maps with different keys?

Reply:
I just say it can be easier to have 2 map (first by flowId, second by requestId). I don't say it need to be reworked, just suppose to think about such implementation.